packages/tc-api-rest-handlers/lib/events/__tests__/events-integration.spec.js

```diff
@@ -226,6 +225,31 @@ describe('Rest events module integration', () => {
 			]);
 			expectUpdateEvent(childType, childCode, ['isChildOf']);
 		});
+
+		it('should not send extra UPDATE events when removing rich relationship property that was already non-existent', async () => {
+			const [main, child] = await Promise.all([
+				createMainNode(),
+				createNode(childType, { code: childCode }),
+			]);
```

Comment:
Create some fixture records - a `MainType` (`createMainNode() is a shorthand for `createNode('MainType', { code: mainCode })`... maybe that's a bit too DRY??